deepview_profile/tracking/call_stack.py

```diff
@@ -1,12 +1,19 @@
 import collections
 import inspect
 import os
-
+import re
 import torch
 
 SourceLocation = collections.namedtuple(
     'SourceLocation', ['file_path', 'line_number', 'module_id'])
 
+pattern_list = ["./transformers/models[/\w+/]+\w+.py","./diffusers/models[/\w+/]+\w+.py"]
```

Comment:
Consider moving to a different file for better visibility and organization as this list might grow in the future.

Reply:
moved to utils